src/core.js

```diff
@@ -917,6 +917,21 @@ $.extend( $.validator, {
 				valid: true,
 				message: this.defaultMessage( element, "remote" )
 			});
+		},
+
+		// cleans up all forms and elements, removes validator-specific events
+		destroy: function() {
+			this.resetForm();
+
+			$( this.currentForm )
+				.off( "submit.validate" )
+				.off( "focusin.validate focusout.validate keyup.validate",
```

Comment:
I've just confirmed this with JSFiddle, and yeah, I never expected that to work. Changes pushed and synced.